TheMovieDb/Components/MovieDetail/RecommendedTableViewCell.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  RecommendedTableViewCell.swift
+//  TheMovieDb
+//
+//  Created by Juan David Torres on 04/11/21.
+//
+
+import UIKit
+
+final class RecommendedTableViewCell: UITableViewCell, UICollectionViewDataSource, UICollectionViewDelegate {
+  
+  @IBOutlet weak var typeLabel: UILabel?
+  @IBOutlet weak var collectionView: UICollectionView?
```

Comment:
try to use Compositional layout yo avoid have a collectionview inside a cell inside a tableview